src/platform-implementation-js/views/compose-view.ts

```diff
@@ -443,6 +443,9 @@ export default class ComposeView extends EventEmitter {
 
   public registerRequestModifier(
     modifier: (composeParams: {
+      to: Contact[] | null;
```

Comment:
I think this is a similar case of Flow types not matching the TS.

https://github.com/InboxSDK/InboxSDK/pull/802/files#r1003607449

Reply:
changed flow types to be same as ts types https://github.com/InboxSDK/InboxSDK/pull/802/commits/367459cff8223662ce76d2b349872d9ed84e7bec